src/C1_verification.py

```diff
@@ -75,6 +76,34 @@ def lookup_file(file_path, name):
     return
 
 
+def check_feedin_tariff(dict_values):
+    r"""
+    Raises error if feed-in tariff > energy price of any asset in 'energyProvider.csv'.
+
+    Parameters
+    ----------
+    dict_values : dict
+        Contains all input data of the simulation.
+
+    Returns
+    -------
+    Indirectly, raises error message in case of feed-in tariff > energy price of any
+    asset in 'energyProvider.csv'.
+
+    """
+    for provider in dict_values["energyProviders"].keys():
+        feedin_tariff = dict_values["energyProviders"][provider]["feedin_tariff"][
+            "value"
+        ]
+        electricity_price = dict_values["energyProviders"][provider]["energy_price"][
+            "value"
+        ]
+        if feedin_tariff > electricity_price:
+            msg = f"Feed-in tariff > energy price of energy provider asset '{dict_values['energyProviders'][provider]['label']}' would cause an unbound solution."
```

Comment:
### 
```suggestion
            msg = f"Feed-in tariff > energy price of energy provider asset '{dict_values['energyProviders'][provider]['label']}' would cause an unbound solution and terminate the optimization."
```